packages/server/src/serverConnection.ts

```diff
@@ -55,47 +55,78 @@ export default class ServerConnection implements IServerConnection {
     inputs: Inputs,
     token: CancellationToken
   ): Promise<Result<string, FxError>> {
-    const res = await this.core.createProject(inputs);
+    const corrId = inputs.correlationId ? inputs.correlationId : "";
+    const res = await Correlator.runWithId(
+      corrId,
+      (params) => this.core.createProject(params),
+      inputs
+    );
     return standardizeResult(res);
   }
 
   public async localDebugRequest(
     inputs: Inputs,
     token: CancellationToken
   ): Promise<Result<Void, FxError>> {
-    const res = await this.core.localDebug(inputs);
+    const corrId = inputs.correlationId ? inputs.correlationId : "";
+    const res = await Correlator.runWithId(
+      corrId,
+      (params) => this.core.localDebug(params),
+      inputs
+    );
     return standardizeResult(res);
   }
 
   public async provisionResourcesRequest(
     inputs: Inputs,
     token: CancellationToken
   ): Promise<Result<Void, FxError>> {
-    const res = await this.core.provisionResources(inputs);
+    const corrId = inputs.correlationId ? inputs.correlationId : "";
+    const res = await Correlator.runWithId(
+      corrId,
+      (params) => this.core.provisionResources(params),
+      inputs
+    );
+    console.log(res);
```

Comment:
Can comment/delete this log? it makes this method more different from other methods

Reply:
oh, sure. I forgot it.